src/js/serial.js

```diff
@@ -258,13 +266,21 @@ const serial = {
         }
     },
     getDevices: function (callback) {
+        const self = this;
+
         chrome.serial.getDevices(function (devices_array) {
             const devices = [];
             devices_array.forEach(function (device) {
-                devices.push({
-                              path: device.path,
-                              displayName: device.displayName,
-                             });
+                const isFC = self.serialDevices.some(el => el.vendorId === device.vendorId) && self.serialDevices.some(el => el.productId === device.productId);
```

Comment:
`isFC` isn't true, not all USB to UART devices have an FC attached. Perhaps `isKnownSerialDevice` ?

Reply:
Changed